File: midai-pay-posw-api/src/main/java/com/midai/pay/routetransactionInformation/TransactionInformation.java
package com.midai.pay.routetransactionInformation;

import java.util.Date;

import lombok.Data;

@Data
public class TransactionInformation {

	
	/*** 商户手机号码  */
	private String moble;
	/*** 交易金额  */
	private Double money;
	/*** 交易卡号 */
	private String cardNo;
	/*** 交易时间 */
	private Date transcationTime;
	/*** 交易区域 */
	private String areaCode;
	
}

File: midai-pay-posw-service/src/main/resources/roleconfig/TransactionInformation.py
class TransactionInformation(): 
 
   def __init__(self):
       moble = ""
       money = 0
       cardNo = ""
       transcationTime = 0
       areaCode = ""
       
   def setMoble(self, moble): 
       self.moble = moble
       
   def setMoney(self, money): 
       self.money = money
       
   def setCardNo(self, cardNo): 
       self.cardNo = cardNo
       
   def setTranscationTime(self, transcationTime): 
       self.transcationTime = transcationTime
       
   def setAreaCode(self, areaCode): 
       self.areaCode = areaCode
       
   def getChannelCodes(self):
       if self.money >= 5000*100 : 
           return '301000000001'
       else :
           return '00000081'
       